main.md

```diff
@@ -193,180 +199,327 @@ Note: Applications MUST ensure that their authorization data types do not collid
 The following example shows how an implementation could utilize the namespace `https://scheme.example.org/` to ensure collision resistant element names.
 
 ```JSON
-{ 
-   "type":"https://scheme.example.org/files",
-   "locations":[ 
+{
+   "type": "https://scheme.example.org/files",
+   "locations": [
       "https://example.com/files"
    ],
-   "permissions":[ 
-      { 
-         "path":"/myfiles/A",
-         "access":[ 
+   "permissions": [
+      {
+         "path": "/myfiles/A",
+         "access": [
             "read"
          ]
       },
-      { 
-         "path":"/myfiles/A/X",
-         "access":[ 
+      {
+         "path": "/myfiles/A/X",
+         "access": [
             "read",
             "write"
          ]
       }
    ]
 }
 ```
+## Relationship to "scope" parameter {#scope}
+
+`authorization_details` and `scope` can be used in the same authorization request for carrying independent authorization requirements. 
+
+The AS MUST keep both requirements sets separate when reflecting this data to the client and passing this data through to resource servers.
```

Comment:
Why is this a requirement? It might make sense for the AS to treat a `scope` value as a shorthand for one of the `authorization_details` objects, or even a set of them. I think it's more important that the AS always merge them together to create the whole request. In other words, it's an AND.

Reply:
Do you intend to define a data structure for conveying the merged result?

I tend towards keeping the way scopes are passed through in JWTs and Introspection Responses as is and define an additional pipe for authorization details.

Reply:
It was not my intent to define a merged data structure, but having guidance for how to combine these two is going to be one of the harder parts of merging this in with OAuth 2's legacy code.

Reply:
Can you please propose text?

Reply:
```suggestion
The AS MUST consider both sets of requirements in combination with each other for the given authorization request. The details of how the AS combines these parameters are specific to the APIs being protected and outside the scope of this specification.

It is RECOMMENDED that a given API use only one form of requirement specification. 
```